packages/desktop-client/src/components/reports/ReportSidebar.jsx

```diff
@@ -96,6 +97,9 @@ export function ReportSidebar({
       if (customReportItems.graphType === 'StackedBarGraph') {
         setGraphType('BarGraph');
         graph = 'BarGraph';
+      } else if (['LineGraph'].includes(customReportItems.graphType)) {
+        setGraphType('TableGraph');
+        onChangeViews('viewLegend', false);
```

Comment:
This should not be needed if you modify the disabledlists.ts file properly.
https://github.com/actualbudget/actual/blob/38ffccb9037f6ea0f233cdf5d36f72bb8423f13b/packages/desktop-client/src/components/reports/disabledList.ts#L4